crates/jstz_cli/src/config.rs

```diff
@@ -243,12 +274,52 @@ impl Config {
 
     pub fn octez_rollup_node(&self) -> Result<OctezRollupNode> {
         let sandbox = self.sandbox()?;
+        let network = self.current_network()?;
 
         Ok(OctezRollupNode {
             octez_rollup_node_bin: Some(self.octez_path.join("octez-smart-rollup-node")),
             octez_rollup_node_dir: sandbox.octez_rollup_node_dir.clone(),
             octez_client_dir: sandbox.octez_client_dir.clone(),
-            endpoint: format!("http://127.0.0.1:{}", sandbox.octez_node_rpc_port),
+            endpoint: network.octez_node_rpc_endpoint,
         })
     }
+
+    /// set the network to use, if the network is None, use the default local network used in sandbox.
+    fn set_network(&mut self, network: Option<String>) -> io::Result<()> {
+        match network {
+            None => {
+                self.networks.current_network = None;
+                println!("Using local network used in sandbox");
+                Ok(())
+            }
+            Some(network) => {
+                // the network must be specified in .json/config.json
+                if self.networks.networks.contains_key(&network) {
+                    println!("Using network `{}`", network);
+                    self.networks.current_network = Some(network);
+                    return Ok(());
+                }
+
+                Err(Error::new(ErrorKind::InvalidData, format!(
+                    "Invalid network: {}, please specify the network in .json/config.json",
+                    &network)))
+            }
+        }
+    }
+
+    /// get the selected network, if the current_network is None, use the default local network used in sandbox
```

Comment:
A very small nit - but there should be consistency in the first letter case in comments. Other comments use capital case, so it would be better to use it also here.